src/wp-includes/interactivity-api/interactivity-api.php

```diff
@@ -0,0 +1,134 @@
+					remove_filter( 'render_block', $process_interactive_blocks );
+					$root_interactive_block = null;
+				}
+				return $content;
+			};
+
+			/*
+				* Uses a priority of 20 to ensure that other filters can add additional
+				* directives before the processing starts.
+				*/
+			add_filter( 'render_block', $process_interactive_blocks, 20, 2 );
+		}
+	}
+
+	return $parsed_block;
+}
+add_filter( 'render_block_data', 'wp_interactivity_process_directives_of_interactive_blocks', 10, 1 );
+
+/**
+ * Retrieves the main WP_Interactivity_API instance.
+ *
+ * It provides access to the WP_Interactivity_API instance, creating one if it
+ * doesn't exist yet. It also registers the hooks and necessary script
+ * modules.
+ *
+ * @since 6.5.0
+ *
+ * @return WP_Interactivity_API The main WP_Interactivity_API instance.
+ */
+function wp_interactivity() {
+	static $instance = null;
+	if ( is_null( $instance ) ) {
+		$instance = new WP_Interactivity_API();
+		$instance->add_hooks();
+		$instance->register_script_modules();
+	}
+	return $instance;
+}
+
+/**
+ * Processes the interactivity directives contained within the HTML content
+ * and updates the markup accordingly.
+ *
+ * @since 6.5.0
+ *
+ * @param string $html The HTML content to process.
+ * @return string The processed HTML content. It returns the original content when the HTML contains unbalanced tags.
+ */
+function wp_interactivity_process_directives( $html ) {
+	return wp_interactivity()->process_directives( $html );
+}
+
+/**
+ * Gets and/or sets the initial state of an Interactivity API store for a
+ * given namespace.
+ *
+ * If state for that store namespace already exists, it merges the new
+ * provided state with the existing one.
+ *
+ * @since 6.5.0
+ *
+ * @param string $store_namespace The unique store namespace identifier.
+ * @param array  $state           Optional. The array that will be merged with the existing state for the specified
+ *                                store namespace.
+ * @return array The current state for the specified store namespace.
+ */
+function wp_interactivity_state( $store_namespace, $state = null ) {
+	return wp_interactivity()->state( $store_namespace, $state );
+}
+
+/**
+ * Gets and/or sets the configuration of the Interactivity API for a given
+ * store namespace.
+ *
+ * If configuration for that store namespace exists, it merges the new
+ * provided configuration with the existing one.
+ *
+ * @since 6.5.0
+ *
+ * @param string $store_namespace The unique store namespace identifier.
+ * @param array  $config          Optional. The array that will be merged with the existing configuration for the
+ *                                specified store namespace.
+ * @return array The current configuration for the specified store namespace.
+ */
+function wp_interactivity_config( $store_namespace, $initial_state = null ) {
+	return wp_interactivity()->config( $store_namespace, $initial_state );
```

Comment:
It looks like the `$config` variable here was misnamed. I'd also make it clear we're returning the new config after dealing with it (current is confusing for me here):

```suggestion
 * @param string $store_namespace The unique store namespace identifier.
 * @param array  $config          Optional. The array that will be merged with the existing configuration for the
 *                                specified store namespace.
 * @return array The configuration for the specified store namespace. This will be the updated configuration if a $config argument was provided.
 */
function wp_interactivity_config( $store_namespace, $config = null ) {
	return wp_interactivity()->config( $store_namespace, $config );
```

Reply:
Thanks! Fixed in https://github.com/WordPress/wordpress-develop/pull/5953/commits/7aaf308be82ac20be5e182d4a351a626b249e6f2.